src/ckb-daemon/led_keyboard.c

```diff
@@ -71,6 +71,17 @@ static void makergb_full(const lighting* light, uchar data_pkt[12][MSG_SIZE]){
     memcpy(data_pkt[10] + 4, b + 120, 60);
 }
 
+static void makergb_k55(const lighting* light, uchar data_pkt[MSG_SIZE]){
+    const uchar* r = light->r, *g = light->g, *b = light->b;
+    // The K55 uses RGBRGBRGB colouring.
+    for (int i = 0; i < 3; i++) {
+        int index = (i * 3) + 4;
+        data_pkt[index + 0] = r[i];
+        data_pkt[index + 1] = g[i];
+        data_pkt[index + 2] = b[i];
+    }
+}
+
 static int rgbcmp(const lighting* lhs, const lighting* rhs){
     // Compare two light structures, ignore mouse zones
     return memcmp(lhs->r, rhs->r, N_KEYS_HW) || memcmp(lhs->g, rhs->g, N_KEYS_HW) || memcmp(lhs->b, rhs->b, N_KEYS_HW);
```

Comment:
This compares neither the platinum top bar, nor the generic zones for keyboards. We should either extend the memcpy, or make a "hole" somehow.